### PR TITLE
Identifiers Name Change in Vietnam

### PR DESCRIPTION
**Pull request by ClipperWings**
VCL_CTR added
VVVV to VVHN,
VVTS to VVHM,
VDPP to VDPF and ASEA added in Hong Kong
Ref #249

### CODE REVIEW

File: VATSpy.dat [hunk 1]
```diff
@@ -18385,9 +18385,9 @@ ADR_U|Adria Radar Upper|LJLA,LDZO,LQSB,LYBA,LWSS,LAAA
 AFRN|North Africa Control|DAAA,DRRR,DTTC,FTTT,GMMM,HLLL
 AFRE|East Africa Control|HAAA,HCSM,HHAA,HSSS
 AFRS|Africa South Control|FACA,FAJA,FBGR,FIMM,FLFI,FMMM,FNAN,FQBE,FVHF,FWLL,FYWH,HKNA,HTDC,HUEC
-ASEA_N|South East Asia Control|RPHI,VVTS,VTBB,VVVV,VLVT,VDPP
+ASEA_N|South East Asia Control|VHHK,RPHI,VVHM,VTBB,VVHN,VLVT,VDPF
 ASEA_S|South East Asia Control|WSJC,WMFC,WBFC,WIIF,WAAF
-ASEA|South East Asia Control|WSJC,WMFC,WBFC,WIIF,WAAF,RPHI,VVTS,VDPP,VTBB,VVVV,VLVT,VYYF
+ASEA|South East Asia Control|VHHK,WSJC,WMFC,WBFC,WIIF,WAAF,RPHI,VVHM,VDPF,VTBB,VVHN,VLVT,VYYF
```
Comment (Adrian2k) [inline VATSpy.dat:18390]: You seem to have added VHHK, which seems unrelated to the Vietnam changes.

Is there any updated documentation/information on what ASEA covers?
According to the website, ASEA covers neither VHHK or VYYF - https://www.vat-sea.com/asea

**Review by Adrian2k — COMMENTED** [review]
Looking good, however something that caught my eye:

File: VATSpy.dat (hunk 1 above)
Comment (ClipperWings) [reply VATSpy.dat:18390]: Hi Adrian,

I'm also the director for ASEA_FSS
You can see in the link. We're in process of adding it in and it's been OKayed by the vACC. Sorting out the kinks and docs now.

Cheers,
David.

**Review by Adrian2k — APPROVED** [review]

**Review by NelisV — APPROVED** [review]
